rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/schedule/Sw360ScheduleService.java

```diff
@@ -42,4 +41,84 @@ public RequestStatus cancelAllServices(User sw360User) throws TException {
         }
     }
 
+    public RequestSummary scheduleCveSearch(User sw360User) throws TException{
+        String serviceName = ThriftClients.CVESEARCH_SERVICE;
+        try {
+            if (PermissionUtils.isUserAtLeast(UserGroup.ADMIN, sw360User)) {
+                RequestSummary requestSummary = new ThriftClients().makeScheduleClient().scheduleService(serviceName);
+                return requestSummary;
+            } else {
+                throw new HttpMessageNotReadableException("User is not admin");
+            }
+        } catch (TException e) {
+            throw new TException(e.getMessage());
+        }
+    }
+
+    public RequestStatus cancelCveSearch(User sw360User) throws TException{
+        String serviceName = ThriftClients.CVESEARCH_SERVICE;
+        try {
+            if (PermissionUtils.isUserAtLeast(UserGroup.ADMIN, sw360User)) {
+                RequestStatus requestStatus = new ThriftClients().makeScheduleClient().unscheduleService(serviceName, sw360User);
+                return requestStatus;
+            } else {
+                throw new HttpMessageNotReadableException("User is not admin");
+            }
+        } catch (TException e) {
+            throw new TException(e.getMessage());
+        }
+    }
+
+    public RequestSummary deleteAttachmentService(User sw360User) throws TException{
+        try {
+            if (PermissionUtils.isUserAtLeast(UserGroup.ADMIN, sw360User)) {
+                RequestSummary requestSummary = new ThriftClients().makeScheduleClient().scheduleService(ThriftClients.DELETE_ATTACHMENT_SERVICE);
```

Comment:
This can be extracted to a method as this is used in many places `new ThriftClients().makeScheduleClient().scheduleService(_)` as well as

Reply:
done.